### PR TITLE
Scp1344Item & move Prefabs

### PR DESCRIPTION
**Pull request by Misfiy**
## Description
**Describe the changes** 
* Remove ToysHelper, put Prefabs into the class theyre prefabs of
* Add Scp1344 item wrapper

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?** (if this is a feature change)

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

**Other information**:

<br />

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentations
<br />

## Submission checklist

- [ ] I have checked the project can be compiled
- [ ] I have tested my changes and it worked as expected

### Patches (if there are any changes related to Harmony patches)
- [ ] I have checked no IL patching errors in the console

### Other
- [X] Still requires more testing

### CODE REVIEW

**Comment by Misaka-ZeroTwo** [comment]
build error moment

**Comment by Misfiy** [comment]
> build error moment

forgot to commit ooops

**Review by VALERA771 — APPROVED** [review]

File: EXILED/Exiled.API/Features/Toys/Speaker.cs [hunk 1]
```diff
@@ -87,12 +87,25 @@ public float MinDistance
...
+        /// Creates a new <see cref="Speaker"/>.
         /// </summary>
-        /// <returns>The <see cref="Speaker"/>.</returns>
-        public static Speaker Create()
+        /// <param name="position">The position of the <see cref="Speaker"/>.</param>
+        /// <param name="rotation">The rotation of the <see cref="Speaker"/>.</param>
+        /// <param name="scale">The scale of the <see cref="Speaker"/>.</param>
+        /// <param name="spawn">Whether the <see cref="Speaker"/> should be initially spawned.</param>
+        /// <returns>The new <see cref="Speaker"/>.</returns>
+        public static Speaker Create(Vector3? position, Vector3? rotation, Vector3? scale, bool spawn)
```
Comment (VALERA771) [inline EXILED/Exiled.API/Features/Toys/Speaker.cs:97]: if you consider them as nullable maybe add a `null` default value?

Comment (BoltonDev) [reply EXILED/Exiled.API/Features/Toys/Speaker.cs:97]: we can add default value yeah, i just copied the method from another toy

**Comment by BoltonDev** [comment]
Don't merge this pr for the moment because there is a problem with the prefab loader. We actually load them from NetworkClient which doesn't register prefabs before map generation so actually if we push this, we're unable to spawn things that use prefab when the map is generated

**Review by louis1706 — APPROVED** [review]

**Review by BoltonDev — APPROVED** [review]

**Comment by Misaka-ZeroTwo** [comment]
There are conflicts.

**Pull request merged**